File: ListaAdapter/app/src/main/java/br/harlan/listaadapter/MainActivity.java
package br.harlan.listaadapter;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.ArrayAdapter;
import android.widget.ListView;
import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    ArrayList<Pessoa> lista;
    ArrayAdapter<Pessoa> adaptadorLista;
    ListView listView;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        listView=(ListView)findViewById(R.id.listView);
        lista=new ArrayList<Pessoa>();
        preencherArrayList();
        adaptadorLista=new ArrayAdapter<Pessoa>(this, android.R.layout.simple_list_item_1,lista);
        listView.setAdapter(adaptadorLista);
    }

    public void preencherArrayList(){
        Pessoa pessoa=new Pessoa("Harlan","123");
        lista.add(pessoa);
        pessoa.nome="Vanessa";
        pessoa.cpf="234";
        lista.add(pessoa);
        pessoa.nome="vinicius";
        pessoa.cpf="345";
        lista.add(pessoa);
    }
}

File: AsyncTask/app/src/main/java/br/com/harlan/asynctask/MainActivity.kt
package br.com.harlan.asynctask

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var intent: Intent = Intent(this, CadastrarUsuarioActivity::class.java)
        startActivity(intent)
    }
}
